pages/about.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import About from './about.page';
+
+describe('<About />', () => {
+  it('should render without crashing', async () => {
```

Comment:
[nit] unnecessary async 😉 
```suggestion
  it('should render without crashing', () => {
```